### ISSUE TITLE
An error occurs when a subdirectory exists in the image directory

### ISSUE THREAD

**Issue by dash14** [sonodar/reg-publish-filesystem-plugin#3]
Hi @sonodar,
Thank you for publishing this great plugin that enables us to easily use reg-suit locally!

In our test environment, the directory where captured images are stored is hierarchical. In other words, it contains subdirectories.
When using reg-publish-filesystem-plugin in that environment, I encountered the following error.

```
$ npx reg-suit run
[reg-suit] info version: 0.14.3
[reg-suit] info Detected the previous snapshot key: 'xxxxxxxxxxxxxxxx'
                                         ■ 0% | ETA: 0s | 0/1[reg-publish-filesystem-plugin] info Download 1 files from .reg-assets.
SystemError [ERR_FS_EISDIR]: Path is a directory: cp returned EISDIR (/Users/dash14/example/.reg-assets/xxxxxxxxxxxxxxxx/actual/sub_dir is a directory (not copien
    at new SystemError (node:internal/errors:250:5)
    at new NodeError (node:internal/errors:361:7)
    at getStatsForCopy (node:internal/fs/cp/cp:200:11) {
  code: 'ERR_FS_EISDIR',
  info: {
    message: '/Users/dash14/example/.reg-assets/xxxxxxxxxxxxxxxx/actual/sub_dir is a directory (not copied)',
    path: '/Users/dash14/example/.reg-assets/xxxxxxxxxxxxxxxx/actual/sub_dir',
    syscall: 'cp',
    errno: 21,
    code: 'EISDIR'
  },
  errno: [Getter/Setter: 21],
  syscall: [Getter/Setter: 'cp'],
  path: [Getter/Setter: '/Users/dash14/example/.reg-assets/xxxxxxxxxxxxxxxx/actual/sub_dir']
}
```

The reg-suit supports a hierarchical directory structure.
It would be very helpful if this plugin could also handle the existence of subdirectories without problems.
I will create a PR after this. I would be pleased if you could review it.

**Comment by sonodar** [comment]
@dash14 
This issue has been resolved in 0.2.0.  
https://www.npmjs.com/package/reg-publish-filesystem-plugin/v/0.2.0

**Comment by dash14** [comment]
@sonodar
Thanks for the quick release!
Once again, I appreciate your creation of such a great plugin!
